config/config-network.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
   labels:
     serving.knative.dev/release: devel
   annotations:
-    knative.dev/example-checksum: "3317ccca"
+    knative.dev/example-checksum: "2653766a"
```

Comment:
Just to check: if this isn't updated, will the tests fail?

Reply:
Didn't fail in my PR.

Reply:
Dunno, do we call verify codegen?